### PR TITLE
Added humane-js log message system. Made whitespace changes.

### PR DESCRIPTION
**Pull request by dghalbr**

### CODE REVIEW

**Comment by neighbor983** [comment]
I am not seeing the messages. I just the same the same alert messages

**Comment by neighbor983** [comment]
Ok now i see the error message on login. i just needed to refresh the browser.

**Comment by dghalbr** [comment]
I believe I have correctly resolved the conflicts.

File: gulpfile.js [hunk 1]
```diff
@@ -54,34 +54,34 @@ gulp.task('fontAwesome', ['clean:fonts'], function () {
...
+                    'public/*.html',
+                    'public/app/**.*html',
+                    'public/app/**/**.*html',
+                    '!public/assets/libs/**.*html'
+                ]
+            }))
+            .pipe(concat('single.css'))
+            .pipe(cssmin())
+            .pipe(rename('app.min.css'))
+            .pipe(gulp.dest('public/dist'));
```
Comment (neighbor983) [inline gulpfile.js]: You are missing so the 'css' is never closed
});

**Comment by neighbor983** [comment]
When i try to test the messaging system i get an error on the registration page i get "TypeError: humane is not a function" in the console.

The message  when i test get when i test on the login page just says undefined.

**Comment by dghalbr** [comment]
I have no idea why there were conflicts all of a sudden out of no where. When I made this pull request there were none and it all worked. I'm going to start over.

**Comment by neighbor983** [comment]
Still getting the "TypeError: humane is not a function" when i try and use an username or email that has already been registered.

**Pull request merged**